MAUI/Segmented-Control/accessibility.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Accessibility in .NET MAUI Segmented Control | Syncfusion
+description: Learn here about Accessibility support with the Syncfusion .NET MAUI Segmented Control (SfSegmentedControl).
+platform: maui
+control: SfSegmentedControl
+documentation: ug
+---
+
+## Accessibility in .NET MAUI Segmented Control (SfSegmentedControl)
+
+The Segmented Control is designed to work effectively with the segment elements, providing voice descriptions of their segment items.
+
+## Keyboard shortcuts in .NET MAUI Segmented Control (SfSegmentedControl)
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Right arrow<br/><br/></td><td>Moves to the next item on the segmented control's right side. It does not make any selection and only navigates to the next item. If the focus border is on the last item, pressing the RightArrow does nothing.<br/><br/></td></tr>
+<tr>
+<td>
+Left arrow<br/><br/></td><td>
+Moves to the previous item on the segmented control's left side. It does not make any selection and only navigates to the previous item. If the focus border is on the first item, pressing the LeftArrow does nothing.<br/><br/></td></tr>
+<tr>
+<td>
+Enter<br/><br/></td><td>
+Pressing the Enter key when segment item in focus will make selection.<br/><br/></td></tr>
```

Comment:
Pressing the Enter key when the segment item is in focus will make a selection.<br/><br/></td></tr>

Reply:
Changed.